Resources/Prototypes/_Arc/Entities/clothing/eyes/arcglasses.yml

```diff
@@ -0,0 +1,91 @@
+#\Resources\Textures\_Arc\Clothing\eyes
+#for my ass not knowing what the directory for this shit
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesMedical
+  name: medical glasses
+  description: Upgraded sunglasses that provide flash immunity and a medical HUD.
+  components:
+  - type: Sprite
+    sprite: _Arc/Clothing/eyes/medglasses.rsi
+  - type: Clothing
+    sprite: _Arc/Clothing/eyes/medglasses.rsi
+  - type: FlashImmunity
+  - type: EyeProtection
+    protectionTime: 5
+  - type: Tag
+    tags:
+    - HamsterWearable
+    - WhitelistChameleon
+    #- SecDogWearable # DeltaV - let Laika wear secglasses (no i dont wanna do another sprite shit for doggo)
+  - type: IdentityBlocker
+    coverage: EYES
+  - type: ShowHealthBars
+    damageContainers:
+    - Biological
+  - type: ShowHealthIcons
+    damageContainers:
+    - Biological
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesMedSec
+  name: medsec glasses
+  description: Upgraded sunglasses that provide flash immunity and a medical HUD and security HUD.
```

Comment:
i prefer to call it medical HUD and security HUD. yeah i dont like putting medsec hud because it sounded lazy as fuck. beside it was almost the same as medsec hud description so yeah im not changing this one.